strongbox-security/strongbox-authentication-registry/src/test/java/org/carlspring/strongbox/authentication/ConfigurableProviderManagerTest.java

```diff
@@ -82,6 +84,7 @@ public void testApplicationContext()
     }
     
     @Test
+    @Disabled
```

Comment:
@anki2189 why `Disabled`?

Reply:
@sbespalov This test is failing . From my initial analysis it is related to concurrency but I am not able fix it as of now.

Reply:
@sbespalov : Would you be able to have a look at it?

Reply:
I could have a look later if there will be an issue.

Reply:
@anki2189 could you please create an issue?

Reply:
Done.